docs/screens/select-screen.md

```diff
@@ -0,0 +1,111 @@
+const transitions = [
+    ...
+    {
+        name: "home",
+        state: new Home(),
+        nextScreenName: () => "characterSelect",
+    },
+    {
+        name: "characterSelect",
+        state: new Select(),
+        nextScreenName: () => "game",
+    },
+    ...
+];
+```
+- When running the game, progressing from the home screen should now take the player to the character select screen.
+
+
+## How do I add items to the select screen carousel?
+- Locate the file `config.json` located inside `themes/your-theme/` (where `your-theme` is the name of your theme).
+- Add a `"choices"` array to your select options and fill it with objects containing the keys `main` and `name`. These must be named the same as the key in your `asset-pack-master.json` file (see [below](#how-does-it-find-the-location-of-my-sprite-assets)).
+- `main`: This refers to the image of the thing the player is selecting. For example in a character select screen, this will be the character itself.
+- `name`: This refers to the image of the name of the thing the player is selecting. For example in a character select screen, it will simply be an image of the character's name.
+- For example:
+```
+{
+    "theme": {
+        "characterSelect": {
+            "choices": [
+                {
+                    "main": "dangermouse",
+                    "name": "dangermouseName"
+                },
+                {
+                    "main": "barney",
+                    "name": "barneyName"
+                },
+                {
+                    "main": "jamillah",
+                    "name": "jamillahName"
+                }
+            ]
+        }
+    }
+}
+```
+
+## How does it find the location of my sprite assets?
+
+- Locate the file `asset-pack-master.json` located inside `themes/your-theme/` (where `your-theme` is the name of your theme).
+- Add your sprite locations and preferred key names to this file.
+- For example:
+```
+{
+    ...
+    "characterSelect": [
+        {
+            "type": "image",
+            "key": "dangermouse",
+            "url": "options/dangermouse_sel.png",
+            "overwrite": false
+        },
+        {
+            "type": "image",
+            "key": "dangermouseName",
+            "url": "options/dangermouse_sel_name.png",
+            "overwrite": false
+        },
+        {
+            "type": "image",
+            "key": "barney",
+            "url": "options/barney_sel.png",
+            "overwrite": false
+        },
+        {
+            "type": "image",
+            "key": "barneyName",
+            "url": "options/barney_sel_name.png",
+            "overwrite": false
+        },
+        {
+            "type": "image",
+            "key": "jamillah",
+            "url": "options/jamillah_sel.png",
+            "overwrite": false
+        },
+        {
+            "type": "image",
+            "key": "jamillahName",
+            "url": "options/jamillah_sel_name.png",
+            "overwrite": false
```

Comment:
Could possibly trim down the example to just a two character selection to keep the docs more concise?